README.md

```diff
@@ -177,6 +179,24 @@ console.log(state.data.getCart().email);
 console.log(state.data.getBillingAddress().email);
 ```
 
+#### Passwordless Sign-in
+Customers could sign in using a single-use link sent to their email address. Once they click on the link, they will be redirected back to the website as a signed-in user.
```

Comment:
Instead of `redirected back to the website`, maybe `redirected back to the store`?